src/Process/PlaceOnGroundProcessing.js

```diff
@@ -0,0 +1,126 @@
+import * as THREE from 'three';
+import Coordinates from '../Core/Geographic/Coordinates';
+import Extent from '../Core/Geographic/Extent';
+
+function tileAt(pt, tile) {
+    if (tile.extent) {
+        if (!tile.extent.pointIsInside(pt)) {
+            return undefined;
+        }
+
+        for (const c of tile.children) {
+            const t = tileAt(pt, c);
+            if (t) {
+                return t;
+            }
+        }
+        if (tile.materials[0].textures[0][0].coords.zoom > -1) {
+            return tile;
+        }
+        return undefined;
+    }
+}
+
+let _canvas;
+function _readTextureValueAt(layer, texture, x, y) {
+    if (texture.image.data) {
+        return texture.image.data[y * texture.image.width + x];
+    } else {
+        if (!_canvas) {
+            _canvas = document.createElement('canvas');
+            _canvas.width = 1;
+            _canvas.height = 1;
+        }
+        const ctx = _canvas.getContext('2d');
+        ctx.drawImage(texture.image, x, y, 1, 1, 0, 0, 1, 1);
+        const d = ctx.getImageData(0, 0, 1, 1);
+        return THREE.Math.lerp(
+            layer.materialOptions.colorTextureElevationMinZ,
```

Comment:
See a5b326029171531086ff4be8af286f86dc7c5a6b

(color textures have values between 0 and 1, so to get a float elevation I added min/max z)